kotest-extensions/kotest-extensions-spring/src/jvmMain/kotlin/io/kotest/spring/SpringListener.kt

```diff
@@ -20,6 +20,8 @@ import kotlin.reflect.full.primaryConstructor
 
 object SpringListener : TestListener {
 
+   var ignoreWarning: Boolean = false
```

Comment:
Which warning are we ignoring? Better to place it in the variable name.

Also, could you please add it as a system property as well?

Reply:
A system property is better yes. @fauscik that's my fault for suggesting the var, but I like keeping the var as well.

I would just do something like if (!ignoreWaring || sysprop("kotest.listener.spring.ignore.warnings") == true) { }

Reply:
I think system property makes sense as well as var, so I will keep both.

Reply:
I have added system prop, but I'm wondering, does it make sense to add dependecy on `kotest-common` to have possibility to use `sysprop` instead of `System.getProperty`?

Reply:
kotest-common should be being pulled in anyway, as this depends on kotest-core

Reply:
But I don't think it matters because sysprop is for multi platform and spring is jvm only